src/metafiles/v2.rs

```diff
@@ -63,12 +63,34 @@ impl Crates2Json {
         fs::write(path, &serde_json::to_vec(&self)?)?;
         Ok(())
     }
+
+    pub fn append_to_path(
+        path: impl AsRef<Path>,
+        cvs: CrateVersionSource,
+        info: CrateInfo,
+    ) -> Result<(), Crates2JsonParseError> {
+        let mut c2 = match Self::load_from_path(path.as_ref()) {
```

Comment:
@passcod Same here, is there any mechanism to avoid concurrent appending (e.g. lockfile)?